content/plugin/sdkv2/testing/acceptance-tests/index.mdx

```diff
@@ -85,43 +64,185 @@ func TestAccXxx(*testing.T)
 
 ## Running Acceptance Tests
 
-A `terraform` binary of version 0.12.26 or above must be available on the local system for the acceptance test driver to run, either on the system `$PATH` or supplied via the `TF_ACC_TERRAFORM_PATH` environment variable described below. If you are running acceptance tests in a CI environment, we recommend that you download and install Terraform with a prior build step.
+Verify provider functionality by running acceptance testing, which is typically done via the [`go test`](https://pkg.go.dev/cmd/go/internal/test) command. After ensuring [Requirements](#requirements-and-recommendations) are met, use any environment capable of running `go test`, such as a local workstation [command line](#command-line), or continuous integration runner, such as [GitHub Actions](#github-actions), to run the acceptance tests.
+
+~> **NOTE**: These tests typically create and destroy actual infrastructure resources, possibly incurring expenses during or after the test duration.
+
+### Requirements and Recommendations
```

Comment:
I think since we now have the "ensure the requirements are met" sentence, we can move this. 👍  Will do so after applying some of these great suggestions.